insta/src/App.scss

```diff
@@ -0,0 +1,86 @@
+html,
+body {
+  font-family: 'Neue Helvetica', sans-serif;
+  color: #2B2F3B;
+  font-size: 16px; 
+}
```

Comment:
I see you like using sass. Great work using what you have learned so far to accomplish the task.